.travis/klee.sh

```diff
@@ -208,6 +208,20 @@ else
 
   set +e # We want to let all the tests run before we exit
   lit -v .
+
+  # If metaSMT is the only solver, then rerun tests with non-default metaSMT backends
+  if [ "X${SOLVERS}" == "XmetaSMT" ]; then
+    if [ "X${METASMT_DEFAULT}" != "Xbtor" ]; then
+      lit -v --param klee_opts=-metasmt-backend=btor --param kleaver_opts=-metasmt-backend=btor .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xstp" ]; then  
+      lit -v --param klee_opts=-metasmt-backend=stp --param kleaver_opts=-metasmt-backend=stp .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xz3" ]; then
+      lit -v --param klee_opts=-metasmt-backend=z3 --param kleaver_opts=-metasmt-backend=z3 .
+    fi
+  fi
+
```

Comment:
Thanks for pointing that out! As I would like to see test results for all backends, I'll opt for more complicated checking logic.

For the CMake build, can you suggest how to pull that part out of the else branch? As CMake does not call `lit` directly but generates a custom target, the correct solution would be imho to generate an additional target for each backend in klee/test/CMakeLists.txt...